client/my-sites/hire-a-pro/index.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { navigation, siteSelection, sites } from 'my-sites/controller';
+import { layout } from './controller';
+import { makeLayout, render as clientRender } from 'controller';
+
+export default function() {
+	if ( config.isEnabled( 'manage/hire-a-builder' ) ) {
+		page( '/hire-a-pro', ...[ siteSelection, sites, makeLayout, clientRender ] );
```

Comment:
what's the point of using the spread here? would it not be the same as passing these as a comment-separated list of args?